_schemaV2.graphql

```diff
@@ -10920,6 +10920,7 @@ type EditionSet implements Sellable {
   price: String
   priceDisplay: String
   priceListed: Money
+  priceListedDisplay: String
```

Comment:
This seems to work for edition sets, now we have a diff in how the `saleMessage` is calculated

Will adjust, looks like artwork handles [this in MP directly ](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/index.ts#L1686-L1710)

<img width="1031" height="529" alt="Screenshot 2025-07-22 at 1 13 26 PM" src="https://github.com/user-attachments/assets/556d5895-ee63-421d-a7ac-7cb15694f15e" />